src/main/java/com/minecolonies/coremod/colony/managers/StatisticAchievementManager.java

```diff
@@ -34,14 +34,28 @@
     private int felledTrees       = 0;
     private int plantedSaplings   = 0;
 
+    /**
+     * The colony of the manager.
+     */
+    private final Colony colony;
+
+    /**
+     * Creates the Stat- and AchievementManager for a colony.
+     * @param colony the colony.
+     */
+    public StatisticAchievementManager(final Colony colony)
+    {
+        this.colony = colony;
+    }
+
     /**
      * List of achievements within the colony.
      */
     @NotNull
     private final List<Advancement> colonyAchievements = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)